CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-20.5.2
+
+This edge introduces a number of new features that provide diagnostic
+information for multicluster setups as well as proxy performance
+improvements.
+
+* CLI
+  * Added a section to the `linkerd check` that validates that all
+    clusters part of a multicluster setup have compatible trust anchors
+  * Modified the `export-service` command to work by transforming yaml
```

Comment:
might want to fully qualify this as `linkerd cluster export-service`, especially since cluster and its sub-commands are hidden.